src/main/resources/messages/LibertyBundles.properties

```diff
@@ -50,6 +50,10 @@ liberty.project.file.selection.dialog.title=Liberty project
 liberty.project.file.selection.dialog.message=Select Liberty project for {0}
 liberty.action.selection.dialog.title=Liberty action
 liberty.action.selection.dialog.message=Select Liberty action to run
+liberty.action.selection.dialog.java-home.null=Unable to launch the {0} language server. Could not resolve the java home system property. Make sure the boot Java runtime of the IDE is pointing to a valid JDK of Java {1} or later. Change the boot Java runtime of the IDE as documented here <a href="https://www.jetbrains.com/help/idea/switching-boot-jdk.html">https://www.jetbrains.com/help/idea/switching-boot-jdk.html</a>
```

Comment:
The names of these message properties are confusing. The messages are related to language server startup. This isn't for Liberty actions or a selection dialog.